WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -101,7 +107,14 @@ class BlogDashboardViewModel {
     }
 
     func isQuickActionsSection(_ sectionIndex: Int) -> Bool {
-        return sectionIndex == DashboardSection.quickActions.rawValue
+        let showMigration = MigrationSuccessCardView.shouldShowMigrationSuccessCard && !WPDeviceIdentification.isiPad()
```

Comment:
We could consider putting this check behind a computed property or a function

`MigrationSuccessCardView.shouldShowMigrationSuccessCard && !WPDeviceIdentification.isiPad()`

since it's the same for all callers in the VM and might help clean things up a bit.